editor/plugins/tiles/tile_set_atlas_source_editor.cpp

```diff
@@ -2555,9 +2590,17 @@ TileSetAtlasSourceEditor::TileSetAtlasSourceEditor() {
 	tool_advanced_menu_button->set_flat(true);
 	tool_advanced_menu_button->get_popup()->add_item(TTR("Create Tiles in Non-Transparent Texture Regions"), ADVANCED_AUTO_CREATE_TILES);
 	tool_advanced_menu_button->get_popup()->add_item(TTR("Remove Tiles in Fully Transparent Texture Regions"), ADVANCED_AUTO_REMOVE_TILES);
+	tool_advanced_menu_button->get_popup()->add_item(TTR("Remove Tiles Outside the Texture"), ADVANCED_CLEANUP_TILES);
 	tool_advanced_menu_button->get_popup()->connect("id_pressed", callable_mp(this, &TileSetAtlasSourceEditor::_menu_option));
 	tool_settings->add_child(tool_advanced_menu_button);
 
+	outside_tiles_warning_label = memnew(Label);
+	outside_tiles_warning_label->set_text(TTR("The current atlas source has tiles outside the texture.") + " (?)");
+	outside_tiles_warning_label->set_tooltip_text(TTR("You can clear it using \"Remove Tiles Outside the Texture\" option in 3 dots menu."));
```

Comment:
I think it's technically called a "kebab menu", but I don't know how widespread this term is for neophytes. So "3 dots" might be fine.
```suggestion
	outside_tiles_warning_label->set_tooltip_text(TTR("You can clear it using \"Remove Tiles Outside the Texture\" option in the 3 dots menu."));
```

Reply:
Could possibly also do this if we want to make sure translators don't mistakenly translate this differently in the menu and in the tooltip. But on the other hand it makes the sentence slightly harder to understand when translating.
```suggestion
	outside_tiles_warning_label->set_tooltip_text(vformat(TTR("You can clear it using the \"%s\" option in the 3 dots menu."), TTR("Remove Tiles Outside the Texture")));
```

Reply:
> I think it's technically called a "kebab menu", but I don't know how widespread this term is for neophytes. So "3 dots" might be fine.

There is a vertical ellipsis unicode character: `⋮`
I could use it here.

Reply:
Sounds good.

Reply:
You need to use `U"..."`.